### PR TITLE
 Graphs to help us debug the path and length of electrical cables

### PR DESCRIPTION
**Pull request by felipegtx**
measure

### CODE REVIEW

File: electrical/AweElectricalClusterInfo.cm [hunk 1]
```diff
@@ -125,6 +125,39 @@ public class AweElectricalClusterInfo {
         }
     }
 
+    public bool build2D = false;
```
Comment (mbr4nt) [inline electrical/AweElectricalClusterInfo.cm]: @felipegtx how do you turn this on and off?

Comment (felipegtx) [reply electrical/AweElectricalClusterInfo.cm]: @mbr4nt It's off by default. To turn it on you'd do that locally on your machine. It's just for testing purposes so I don't think it makes sense to have a bolder infrastructure than this.

Comment (felipegtx) [reply electrical/AweElectricalClusterInfo.cm]: @mbr4nt to do so, just set this variable to `true`.

Comment (mbr4nt) [reply electrical/AweElectricalClusterInfo.cm]: cool. I'd suggest calling it debug2D instead.

Comment (felipegtx) [reply electrical/AweElectricalClusterInfo.cm]: Done!

**Pull request merged**